.github/workflows/test.yml

```
@@ -184,7 +184,7 @@ jobs:
             run: echo "SELECT * FROM mysql.slow_log WHERE sql_text LIKE '%oc_mail%' AND sql_text NOT LIKE '%information_schema%'" | mysql -h 127.0.0.1 -u root -pmy-secret-pw
           - name: Print debug logs
             if: ${{ always() }}
-            run: cat nextcloud/data/horde_*.log
+            run: cat nextcloud/data/mail-*-*-imap.log
```

```suggestion
            run: cat nextcloud/data/mail*.log
```

To also print smtp and sieve like before?

---

This would kind of defeat the purpose of per account logging, as all account smtp transmissions would end up in the same file?

---

NVM. I miss understood this, sure I can change this.